### DISCUSSION TITLE
Can I ignore specific lsp roots?

### DISCUSSION THREAD

**Discussion by Demianeen** [LazyVim/LazyVim#2951] [Q&A] [answered]
I have cssls that have cmd to my root directory, so every time I open a file tree in css I get the view from the root dir to the file:

https://github.com/LazyVim/LazyVim/assets/51330172/74942c33-cc6a-4109-b0af-b537c5a1a1d5

Is it possible to ignore cssls location and just use CWD?

**Comment by dpetka2001** [comment, marked as answer]
Isn't `<leader>E` supposed to open the `neo-tree` in the CWD? Only `<leader>e` shows the LSP root. However, do keep in mind that CWD and LSP root might coincide.

**Reply by Demianeen** [reply]
Sorry, I completely forgot about it)
